src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,82 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_LOGIN_SIZE = 6;
+    private static final int MIN_AGE = 18;
+    private static final int MIN_PASSWORD_SIZE = 6;
+    private static final String INVALID_PASSWORD_MESSAGE =
+            "Invalid user's password for registration: ";
+    private static final String INVALID_AGE_MESSAGE =
+            "Invalid user's age for registration: ";
+    private static final String INVALID_LOGIN_MESSAGE =
+            "Invalid user's login for registration: ";
```

Comment:
let's also not create a constant for exception messages to make your code more consistent